gcp-application-cloud-run/README.md

```diff
@@ -0,0 +1,42 @@
+# gcp-application-cloud-run
```

Comment:
Let’s name it to follow the convention for md apps. massdriver-application-gcp-TYPE

we’ll probably out these in dirs in the future [per](https://massdriverworkspace.slack.com/archives/C03HMFZU4UF/p1661448698024519)